backend/frontend/tests/test_card_stop_button.py

```diff
@@ -0,0 +1,106 @@
+    def setUp(self):
+        super().setUp()
+        self.user = UserFactory()
+        self.user.set_password(self.user.email)
+        self.user.save()
+
+        self.do_login(self.user)
+
+    def make_topic_card(self):
+        self.card = AgileCardFactory(
+            content_item=ContentItemFactory(content_type=ContentItem.TOPIC),
+            status=AgileCard.IN_PROGRESS,
+        )
+
+        self.card.assignees.set([self.user])
+
+        # update field needed for an in-progress topic without logging the start event
+        self.card._create_topic_progress_if_not_exists()
+        self.card.topic_progress.start_time = timezone.now()
+
+    def make_project_card(self, project_submission_type):
+        self.card = AgileCardFactory(
+            content_item=ContentItemFactory(
+                content_type=ContentItem.PROJECT,
+                project_submission_type=project_submission_type,
+            ),
+            status=AgileCard.IN_PROGRESS,
+        )
+        self.card.assignees.set([self.user])
+
+    def test_stop_button_moves_topic_card_to_backlog_column(self):
+        self.make_topic_card()
+
+        self.page.click("text=Stop")
+
+        self.page.wait_for_load_state("networkidle")
+
+        backlog_column = self.page.text_content("div#column_RB")
+        ip_column = self.page.text_content("div#column_IP")
+        project_card_title = self.card.content_item.title
+
+        self.assertNotIn(project_card_title, ip_column)
+        self.assertIn(project_card_title, backlog_column)
+
+    def test_topic_stop_button_logs_card_stopped_event(
+        self,
+    ):
+        self.make_topic_card()
+
+        self.page.click("text=Stop")
+
+        self.page.wait_for_load_state("networkidle")
+
+        self.assertEqual(LogEntry.objects.count(), 1)
+        entry = LogEntry.objects.first()
+
+        self.assertEqual(entry.actor_user, self.user)
+        self.assertEqual(entry.effected_user, self.card.assignees.first())
+        self.assertEqual(entry.object_1, self.card.recruit_project)
+        self.assertEqual(entry.object_2, None)
+        self.assertEqual(entry.event_type.name, creators.CARD_STOPPED)
+
+    def test_stop_button_moves_project_card_to_backlog_column(self):
+        self.make_project_card(ContentItem.LINK)
+
+        self.page.click("text=Stop")
+
+        self.page.wait_for_load_state("networkidle")
+
+        backlog_column = self.page.text_content("div#column_RB")
+        ip_column = self.page.text_content("div#column_IP")
+        project_card_title = self.card.content_item.title
+
+        self.assertNotIn(project_card_title, ip_column)
+        self.assertIn(project_card_title, backlog_column)
+
+    def test_project_stop_button_logs_card_stopped_event(
+        self,
+    ):
+        self.make_project_card(ContentItem.LINK)
+
+        self.page.click("text=Stop")
+
+        self.page.wait_for_load_state("networkidle")
+
+        self.assertEqual(LogEntry.objects.count(), 1)
```

Comment:
If we trigger a start then we would expect 2 entries here. We can also filter by event type name and assert an expected count and continue the rest of checks

Reply:
It turned out not to be an issue because the `self.card.start_topic()` method does not trigger the start event to be logged, only the relevant view(`action_start_card`) does that.